core/templates/pages/notifications-dashboard-page/notifications-dashboard-page.controller.ts

```diff
@@ -1,69 +0,0 @@
-// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
All the old .controller.ts files need to be deleted right? I thought github shows that up differently when a file is deleted.

Reply:
Could you make sure that these files don't exist on the repo? (From what I see here, I think the files still exist as empty files which isn't good.)

Reply:
The diff shows these files were deleted:
![Screenshot from 2020-05-21 10-32-01](https://user-images.githubusercontent.com/34922478/82564015-6c014b80-9b4e-11ea-8a52-26ab50304668.png)